src/Core/Content/Product/Cart/ProductCartProcessor.php

```diff
@@ -1,4 +1,6 @@
-<?php declare(strict_types=1);
+<?php
```

Comment:
please move the declare statement back into line 1, that's our code style convention and the linting step in the CI will fail otherwise.